myuw/static/css/mobile.less

```diff
@@ -2115,7 +2139,12 @@ th.mailman-sections-column { width: 99%; }
     margin-bottom: 10px;
 }
 
-
+.card-badge-action-btn { //change to button
```

Comment:
This is redundant, especially if we instead use a card-action-btn class globally that we can use on any page/card.